gax-grpc/src/main/java/com/google/longrunning/OperationsClient.java

```diff
@@ -105,20 +100,11 @@
  * </code>
  * </pre>
  */
-@Generated("by GAPIC")
+@Generated("by GAPIC v0.0.5")
```

Comment:
Thi sis not directly related to this code change, but v0.0.5 - is it really the version?